services/src/messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowResponsePayload.java

```diff
@@ -37,6 +37,9 @@
     @JsonProperty("diverse_with")
     private List<String> diverseWith;
 
+    @JsonProperty("status-details")
```

Comment:
why '-' instead of '_'?

Reply:
This field has been made like other fields in this class.